cheri_compressed_cap_64r.h

```diff
@@ -0,0 +1,190 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2018-2025 Alex Richardson
+ * Copyright (c) 2024 Codasip
+ *
+ * All rights reserved.
```

Comment:
All rights reserved, should it be desired, is normally tied to the specific Copyright line, no?

Reply:
True - copied from somewhere else. Let me fix this.